python/pyspark/pandas/tests/test_dataframe_conversion.py

```diff
@@ -187,7 +188,10 @@ def test_to_json_with_partition_cols(self):
             output_path = "%s/%s/%s" % (self.tmp_dir, partition_path, output_paths[0])
             self.assertEqual("[%s]" % open(output_path).read().strip(), expected)
 
-    @unittest.skip("Pyperclip could not find a copy/paste mechanism for Linux.")
+    @unittest.skipIf(
+        sys.platform == "linux" or sys.platform == "linux2",
+        "Pyperclip could not find a copy/paste mechanism for Linux.",
+    )
```

Comment:
`Pyperclip` doesn't work only on Linux system, so we don't need to skip this in Mac or Window systems.